### PR TITLE
[WIP] Read indicator in 1-to-1 chats 

### LINKED ISSUE #12886: Add read indicator in 1-to-1 chats
As a user
I want to be able to enable read receipts for my messages
So that I can see whether my message has been read by my recipients

### Description

When reading a message, we should notify the other end that the message has been read.
This feature should be disabled by default, and a user should be able to enable it under "Settings"->"Privacy and security".

If the feature is disabled, it should not be sending nor processing read receipts.

This applies for one to one chats only for the time being.

The design is just a change of color in the double tick.

### PR DESCRIPTION
**Pull request by jo-mut**
Read indicator in 1-to-1 chats 

Fixes #12886 

### Summary

This for provides the UI for the user to be able to notify the other when reading a message that its been read

### Review notes
This is an implementation of the frontend functionality to activate and deactivate read-indicator in 1-to-1 chats
The backend which involves creating the functionality with status-go is not yet complete at this point

### Testing notes
Navigate to Privacy and Security
Identify Read Receipts under privacy
Toggle to switch on and off read receipts

#### Platforms

- Android
- iOS
- macOS
- Linux
- Windows

##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test

- Open Status
- ...
- Step 3, etc.

status: ready

### CODE REVIEW

File: src/status_im/signals/core.cljs [hunk 1]
```diff
@@ -67,6 +67,8 @@
       "envelope.expired"   (transport.message/update-envelopes-status cofx (:ids (js->clj event-js :keywordize-keys true)) :not-sent)
       "message.delivered"  (let [{:keys [chatID messageID]} (js->clj event-js :keywordize-keys true)]
                              (models.message/update-db-message-status cofx chatID messageID :delivered))
+     ;; "message.seen"       (let [{:keys [chatID messageID]} (js->clj event-js :keywordize-keys true)]
```
Comment (flexsurfer) [inline src/status_im/signals/core.cljs]: could you elaborate?

Comment (jo-mut) [reply src/status_im/signals/core.cljs]: Yeah! Actually I should have omitted this line of code from this PR. After having a consultation I came to understand that the back-end implementation to process the status of a message as having been seen is not yet implemented and so this code is not necessary until i implement the ability to send a message signal to process the message status as having been seen which would be on status-go

**Comment by flexsurfer** [comment]
@jo-mut could you please make sure this PR includes only related changes, thanks

**Comment by jo-mut** [comment]
cool

File: src/status_im/signals/core.cljs [hunk 2]
```diff
@@ -75,4 +75,4 @@
       "wallet" (ethereum.subscriptions/new-wallet-event cofx (js->clj event-js :keywordize-keys true))
       "local-notifications" (local-notifications/process cofx (js->clj event-js :keywordize-keys true))
       "community.found" (link.preview/cache-community-preview-data (js->clj event-js :keywordize-keys true))
-      (log/debug "Event " type " not handled"))))
+      (log/debug "Event " type " not handled "))))
```
Comment (flexsurfer) [inline src/status_im/signals/core.cljs]: could you revert this file, pls

**Comment by flexsurfer** [comment]
is this PR ready? im not sure I understand how it supposes to work, I can see only settings changes

**Comment by jo-mut** [comment]
Its not ready, thats only the frontend part of it. am working on the backend

**Comment by flexsurfer** [comment]
oh ok, please make it draft, or add [WIP] in the title in that case

**Comment by jo-mut** [comment]
okay

**Comment by flexsurfer** [comment]
hey @jo-mut is this still relevant?

**Comment by jo-mut** [comment]
This awaits status-go implementation for the same functionality.

**Comment by jo-mut** [comment]
I can close it for now and then reopen when am done working on status-go. Is that alright?

**Comment by flexsurfer** [comment]
thats fine to keep it then

**Comment by jo-mut** [comment]
okay thanks

**Pull request closed without merging**